lib/ruby_lsp/setup_bundler.rb

```diff
@@ -24,6 +25,7 @@ def initialize(project_path)
       @custom_dir = T.let(Pathname.new(".ruby-lsp").expand_path(Dir.pwd), Pathname)
       @custom_gemfile = T.let(@custom_dir + "Gemfile", Pathname)
       @custom_lockfile = T.let(@custom_dir + "Gemfile.lock", Pathname)
+      @lockfile_hash = T.let(@custom_dir + "lockfile_hash", Pathname)
```

Comment:
```suggestion
      @lockfile_hash_path = T.let(@custom_dir + "lockfile_hash", Pathname)
```
(I would think of `lockfile_hash` as the string inside the file).

Reply:
Also, perhaps name it `Gemfile.lock.hash` to indicate its correspondence?

Reply:
I updated the ivar name and changed the path to `main_lockfile_hash`. I considered `Gemfile.lock.hash`, but my fear was that it would be mistakenly understood as if it was the hash for the custom lockfile, given that they're both in the same folder.